pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -48,6 +48,24 @@ var (
 	OperatorCephBaseImageVersion string
 )
 
+// CanIgnoreHealthErrStatusInReconcile determines whether a status of HEALTH_ERR in the CephCluster can be ignored safely.
+func CanIgnoreHealthErrStatusInReconcile(cephCluster cephv1.CephCluster, controllerName string) bool {
+	// Get a list of all the keys causing the HEALTH_ERR status.
+	var healthErrKeys = make([]string, 0)
+	for key, health := range cephCluster.Status.CephStatus.Details {
+		if health.Severity == "HEALTH_ERR" {
+			healthErrKeys = append(healthErrKeys, key)
+		}
+	}
+
+	// If MDS_ALL_DOWN is the only cause of HEALTH_ERR we ignore it, do so.
+	var mdsAllDownIgnore = len(healthErrKeys) == 1 && healthErrKeys[0] == "MDS_ALL_DOWN"
```

Comment:
Let's have a slice of codes we want to ignore, so we can more easily add new ones` in the future.

Reply:
Done